models/event.js

```diff
@@ -203,6 +203,8 @@ Event.fromRecord = (record) => {
   }
   const event = new Event()
   event.event_id = decoded[Fields.event_id]
+  // Use collector timestamp as event creation time, forcing UTC
+  event.created_at = new Date(decoded[Fields.collector_tstamp] + '+0000')
```

Comment:
Not sure I understand how appending `+0000` forces UTC.

Reply:
I guess it works though, tried it in the console.

Reply:
The time strings from snowplow say in the docs they are all in UTC, but since they don't include a timezone in the string, JavaScript parses it as the local timezone by default. Adding `+0000` adds a timezone offset of 0, meaning UTC.